### PR TITLE
Kubernetes prerequisites content additions and revisions

### PR DESCRIPTION
**Pull request by bloqhead**
These are changes that were derived from [this PR](https://github.com/Kong/kuma-website/pull/130/files#diff-52e9a2fbf4cf6fe920061fc331b88d6e). I've only made the changes to the Documentation Overview page for v0.3.2. If these changes are good, I can make them across all of the other versions as well.

### CODE REVIEW

**Comment by bloqhead** [comment]
@devadvocado since you were the one that originally worked on these changes, can you review when you have a chance and let me know if I'm on target? I made some revisions based on the feedback provided by @yskopets.

**Comment by devadvocado** [comment]
Looks good to me, thanks for updating the original PR

File: docs/docs/0.3.2/documentation/overview.md [hunk 1]
```diff
@@ -61,6 +61,34 @@ When following the installation instructions, `kuma-injector` will be automatica
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+ ### Matching Labels in `Pod` and `Service`
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.3.2/documentation/overview.md]: There is a space before `###`, won't it break anything?

Comment (bloqhead) [reply docs/docs/0.3.2/documentation/overview.md]: Good catch. I've removed it.

**Pull request merged**